src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X500DirectoryStringHelper.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace System.Security.Cryptography.X509Certificates
 {
-    internal static class X500DictionaryStringHelper
+    internal static class X500DirectoryStringHelper
```

Comment:
Strict, this is not required for pull request, but this file and class were misnamed, so let's fix it.